google-compute-engine/src/main/java/org/jclouds/googlecomputeengine/domain/NewInstance.java

```diff
@@ -81,6 +81,10 @@ public static NewInstance create(String name, URI machineType, URI network, URI
    }
 
    public static NewInstance create(String name, URI machineType, URI network, List<AttachDisk> disks, String description) {
+      return create(name, machineType, network, disks, description, null);
+   }
+
+   public static NewInstance create(String name, URI machineType, URI network, List<AttachDisk> disks, String description, Scheduling scheduling) {
```

Comment:
@nacx This isn't great, but the old factory method is needed to keep this backwards-compatible